src/plugins/discover/public/build_services.ts

```diff
@@ -98,9 +105,11 @@ export function buildServices(
     urlForwarding: plugins.urlForwarding,
     timefilter: plugins.data.query.timefilter.timefilter,
     toastNotifications: core.notifications.toasts,
+    notifications: core.notifications,
     uiSettings: core.uiSettings,
     storage,
     trackUiMetric: usageCollection?.reportUiCounter.bind(usageCollection, 'discover'),
+    triggersActionsUi: plugins.triggersActionsUi,
```

Comment:
the flyout that allows the creation of the rule misses the given `application` and `notifications`.